src/grt/grt-strings.ads

```diff
@@ -30,7 +30,18 @@ package Grt.Strings is
 
    --  Return True IFF C is a whitespace character (as defined in LRM93 14.3)
    function Is_Whitespace (C : in Character) return Boolean;
+   function First_Non_Whitespace_Pos (Str : String) return Integer;
+   function Last_Non_Whitespace_Pos (Str : String) return Integer;
+   function New_Line_Pos (Line : String) return Integer;
+   function Find (Str : String; Char : Character) return Integer;
+   function Find (Str : String; Char : Character; Start : Positive)
+            return Integer;
```

Comment:
Please, add a comment before each specification to explain the purpose of each subprogram.